### ISSUE TITLE
Use Frappe Charts Next

### ISSUE THREAD

**Issue by scmmishra** [mathiasbynens/covid-19-vaccinations-germany#30]
Heya, maintainer of Frappe Charts here. We have v2 for frappe charts in Beta. You can check out the `v2-beta` branch on Github. It has some new features and other enhancements, on of it is better tooltips.

Current tooltips have this scaling problem.

<img width="1440" alt="Screenshot 2021-05-07 at 14 42 56" src="https://user-images.githubusercontent.com/18097732/117427285-850f3200-af42-11eb-8a6f-9df77f5f6442.png">

This is solved in v2 with new tooltips looking like this.

<img width="833" alt="Screenshot 2021-05-07 at 14 34 54" src="https://user-images.githubusercontent.com/18097732/117427356-97896b80-af42-11eb-8206-ceabad064e7f.png">

I do _almost_ weekly releases of v2 beta expecting a official release soon. You it's published under the `next` tag, you can try it out in your project by replacing the CDN link with this

```diff
- import { Chart } from 'https://cdn.jsdelivr.net/npm/frappe-charts@1.5.7/dist/frappe-charts.min.esm.js';
+ import { Chart } from 'https://unpkg.com/frappe-charts@2.0.0-rc14/dist/frappe-charts.esm.js';
```

**Comment by mathiasbynens** [comment]
Will do, thanks for the suggestion! v2 looks awesome and feels faster.

I wonder why the jsdelivr link doesn’t work though. `https://cdn.jsdelivr.net/npm/frappe-charts@2.0.0-rc14/dist/frappe-charts.min.esm.js` is available but doesn’t seem to actually be v2.0.0-rc14?

**Comment by mathiasbynens** [comment]
Live on https://mathiasbynens.github.io/covid-19-vaccinations-germany/. Looks great! Thanks so much <3
